checkpoint.json

```diff
@@ -7,12 +7,12 @@
     "type": "firewall",
     "license": "Copyright (c) 2017-2021 Splunk Inc.",
     "main_module": "checkpoint_connector.py",
-    "app_version": "2.0.0",
-    "utctime_updated": "2018-12-07T01:34:33.000000Z",
+    "app_version": "2.0.4",
```

Comment:
suggestion: Seeing that there are multiple new actions here, should the version be updated to `2.1.0`?
I will leave it up to you.